CONTRIBUTING.md

```diff
@@ -0,0 +1,121 @@
+
+On most projects, we use auto assignment for reviews, but if you want a review from a specific person, feel free to [ask them for a review](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review).
+
+You can expect a review from those assigned within 24-48 hours. Once the reviewers have completed their review, we encourage contributors to adhere to a similar timeframe in addressing the feedback. Once you have addressed the review feedback, you should [re-request a review](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/about-pull-request-reviews#re-requesting-a-review) from the reviewers.
+
+If you find that a pull request has become stalled, feel free to ping the required reviewers using their GitHub username handle. You can also notify the `@mdn/mdn-community-engagement` team using this handle or contact us using [Matrix](https://matrix.to/#/#mdn:mozilla.org).
+
+A reviewer can use GitHub’s suggestions feature when reviewing a pull request. It is the responsibility of the author of the pull request to [accept or reject these suggestions](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request).
+
+This brings up another topic. Suppose the pull request’s author at any time abandons the pull request. In that case, it is the assignee’s responsibility to ensure that the pull request comes to a beneficial conclusion. As a contributor, you can let us know if you can no longer complete the work on the pull request. Responsibility for the pull request then falls on the current assignee.
+
+<!-- TODO: This will be project specific. Sometimes changes might be live instantly or within a few minutes. Update accordingly. -->
+
+Once a pull request is merged, the changes will be live within 24 hours.
+
+## Setting up the development environment
+
+<!-- This is very project specific. The below is merely some guidance that might be applicable for most projects. -->
+
+### Forking and cloning the project
+
+The first step in setting up your development environment is to [fork the repository](https://docs.github.com/en/get-started/quickstart/fork-a-repo) and [clone](https://docs.github.com/en/get-started/quickstart/fork-a-repo#cloning-your-forked-repository) the repository to your local machine.
+
+#### Prerequisites
```

Comment:
Personal preference to have the README list basic local setup steps like in this section (as a kind of "just let me get up and running" thing) have the contributing doc with other gotchas and the lifecycle like you mentioned / what to expect when you open a PR

Reply:
I agree, if the set up is as simple as here, we can add it to the REAME and not here.